src/components/FormAdd.jsx

```diff
@@ -13,10 +18,10 @@ function Form() {
       const selectedOption = selectElement.options[selectElement.selectedIndex];
       dispatch(addBook({
         Type: selectedOption.textContent,
-        Name: document.querySelector('.inpText').value,
+        title: document.querySelector('.inpText').value,
```

Comment:
- [ ] Thi component **Form** should have a local state for _title_ and _author_ the fields. Kindly ensure you use `React.useState` to manage this state with an `onChange` event attached in the React way. Check you previous lesson on react events. 
The component should look something like this 👇🏾 

```javascript
import React from "react"
import { useDispatch } from "react-redux"
import { addBook } from "path/to/booksSlice"

function NewBookForm() {
  const [title, setTitle] = useState("")
  const [author, setAuthor] = useState("")
  const books = useSelector((state) => state.books.books)
  const dispatch = useDispatch()

  // Other logic that controls form state

  return (
    <div>
      {/* Rest of form JSX */}
      <Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
    </div>
  )
}

export default NewBookForm
```